web-src/src/components/ListAlbums.vue

```diff
@@ -145,6 +146,10 @@ export default {
       })
     },
 
+    play_count_changed: function () {
+      this.$emit('play-count-changed')
```

Comment:
this will cause the handler in `page/PagePodcasts.vue` to fire:
```
      <template slot="content">
        <list-albums :albums="albums.items"
            @play-count-changed="reload_new_episodes()"
            @podcast-deleted="reload_podcasts()">
        </list-albums>
```
It could be optmised to send up the `album_id` and have the handler incr all the tracks of given ablum && play_count == 0 instead of asking the server for data again